docs/OpenCLCOnVulkan.md

```diff
@@ -902,3 +904,47 @@ These extension built-in functions are not supported:
 - `sub_group_commit_write_pipe()`
 - `get_kernel_sub_group_count_for_ndrange()`
 - `get_kernel_max_sub_group_size_for_ndrange()`
+
+### Numerical Compliance
+
+Clspv is not able to reach full precision requirements on the supported builtin
```

Comment:
precision -> accuracy

Precision is the number of bits in the representation.  But accuracy tells you how many of them are good.